connector/connect/server/src/test/scala/org/apache/spark/sql/connect/ui/SparkConnectServerListenerSuite.scala

```diff
@@ -37,7 +37,7 @@ class SparkConnectServerListenerSuite
 
   private var kvstore: ElementTrackingStore = _
 
-  private val jobTag = ExecuteJobTag("sessionId", "userId", "operationId")
+  private val jobTag = ExecuteJobTag("userId", "sessionId", "operationId")
```

Comment:
From the code changes, there does seem to be a bug. All the callers are using the correct order. This PR seems just to unify the usages for such functions.

Reply:
The original fix is https://github.com/apache/spark/commit/dd6636b4e076c1c7fd3569ad509d28524a98d156.
I'm fine with it too, but I asked https://github.com/apache/spark/pull/46140#issuecomment-2073342960 because I worried the inconsistency with the similar API.